genesis.json

```diff
@@ -0,0 +1,33 @@
+{
+	"initial_timestamp" : "2017-11-29T10:03:00",
+	"init_supply": 1000,
+	"accounts": [{
+		"name": "initdelegate",
+		"recovery_account": "",
+		"public_key": "STM5omawYzkrPdcEEcFiwLdEu7a3znoJDSmerNgf96J2zaHZMTpWs",
+		"scr_amount": 1000,
+		"sp_ammount": 0
+	}, {
+		"name": "alice",
+		"recovery_account": "",
+		"public_key": "STM5Sg79ehsR4W4XWAbuNkFgmjzWAZQsvgUaP9i1EKVdC6uVbpXTK",
+		"scr_amount": 0,
+		"sp_ammount": 0
+	}, {
+		"name": "bob",
+		"public_key": "STM61EJLQzASMXkZxrNpWB1yhNrCa2T7nUAnFVufoobwvEkNKH798",
+		"votes": 0,
```

Comment:
"votes", "balance" - there are no fields in account_type. What will happen?

Reply:
thx, this is mistake in json.